src/GitVersion.Core.Tests/BuildAgents/GitHubActionsTests.cs

```diff
@@ -96,7 +96,7 @@ public void GetCurrentBranchShouldHandlePullRequests()
         var result = this.buildServer.GetCurrentBranch(false);
 
         // Assert
-        result.ShouldBe("refs/pull/1/merge");
+        result.ShouldBeNull();
```

Comment:
Similar assertions in other build server tests:

https://github.com/GitTools/GitVersion/blob/666bd37b59ae731b1d2bd40b20364b692b21796e/src/GitVersion.Core.Tests/BuildAgents/BuildKiteTests.cs#L75

https://github.com/GitTools/GitVersion/blob/666bd37b59ae731b1d2bd40b20364b692b21796e/src/GitVersion.Core.Tests/BuildAgents/CodeBuildTests.cs#L51

https://github.com/GitTools/GitVersion/blob/666bd37b59ae731b1d2bd40b20364b692b21796e/src/GitVersion.Core.Tests/BuildAgents/SpaceAutomationTests.cs#L87